pkg/cluster/cluster.go

```diff
@@ -610,18 +612,27 @@ func (c *Cluster) initSystemUsers() {
 func (c *Cluster) initRobotUsers() error {
 	for username, userFlags := range c.Spec.Users {
 		if !isValidUsername(username) {
-			return fmt.Errorf("invalid username: '%v'", username)
+			return fmt.Errorf("invalid username: %q", username)
```

Comment:
Shall we briefly mention here how valid usernames look like ?

Reply:
It would be too long to fit into the error message, I think it's a task of the documentation.